VIMVideoPlayer/VIMVideoPlayer.m

```diff
@@ -55,6 +55,8 @@ @interface VIMVideoPlayer ()
 @property (nonatomic, assign) BOOL isTimingUpdateEnabled;
 @property (nonatomic, strong) id timeObserverToken;
 
+@property (nonatomic, strong) AVPlayerItem *item;
```

Comment:
I think we specifically want to hold a strong reference, my understanding is that the KVO crash is occurring because of dissonance between when AVPlayer releases it's `currentItem` and when we have the opportunity to remove observers. So I'd like to hold a strong reference so we have greater control over observer management. Albeit accepting responsibility for keeping our property and the player's item in sync.